src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs

```diff
@@ -138,20 +164,37 @@ async Task<JObject> FindStaticMemberInType(string classNameToFind, string name,
                     {
                         isInitialized = await context.SdbAgent.TypeInitialize(typeId, token);
                     }
-                    var staticFieldValue = await context.SdbAgent.GetFieldValue(typeId, field.Id, token);
-                    var valueRet = await GetValueFromObject(staticFieldValue, token);
-                    // we need the full name here
-                    valueRet["className"] = classNameToFind;
-                    return valueRet;
+                    try
+                    {
+                        var staticFieldValue = await context.SdbAgent.GetFieldValue(typeId, field.Id, token);
+                        var valueRet = await GetValueFromObject(staticFieldValue, token);
+                        // we need the full name here
+                        valueRet["className"] = classNameToFind;
+                        return valueRet;
+                    }
+                    catch (Exception ex)
+                    {
+                        logger.LogDebug(ex, $"Failed to get value of field {field.Name} on {classNameToFind} " +
```

Comment:
I didn't see any thing fail if I removed this catch. Can you add tests to cover this, and the other new try/catch?

Reply:
That's strange. For me without this catch `DebuggerTests.EvaluateOnCallFrameTests.InheritedAndPrivateMembersInAClass(prefix: "")` is failing. It's an old test, not from this PR.